src/Sulu/Bundle/WebsiteBundle/Admin/WebsiteAdmin.php

```diff
@@ -44,4 +67,22 @@ public function getJsBundleName()
     {
         return 'suluwebsite';
     }
+
+    /**
+     * Check the permissions for all webspaces.
+     * Returns true if one webspace has live permission.
```

Comment:
Comment is wrong. And the name of the method doesn't tell too much... What about `checkLivePermission` or even `checkLivePermissionForAllWebspaces`?